assets/stylesheets/bootstrap/_component-animations.scss

```diff
@@ -5,7 +5,7 @@
 // Heads up!
 //
 // We don't use the `.opacity()` mixin here since it causes a bug with text
-// fields in very old browsers. Source: https://github.com/twbs/bootstrap/pull/3552.
+// fields in IE7-8. Source: https://github.com/twbs/bootstrap/pull/3552.
```

Comment:
Please restore the original comment, if someone really wants to know there is a link.  With that said, I do not want to re-introduce speak of IE7-8 into this project.